broadinstitute/rawls
# [qa-1571] upgrade workbench libs and migrate to cats 3

QA-1571

This PR (which includes significant contributions from the CI mob) follows up on these PRs:
* https://github.com/broadinstitute/workbench-libs/pull/770
* https://github.com/broadinstitute/rawls/pull/1547
* https://github.com/broadinstitute/firecloud-develop/pull/2685
* https://github.com/broadinstitute/rawls/pull/1558
And will be completed (hopefully) by this PR:
* https://github.com/broadinstitute/rawls/pull/1556

Changes in this PR
Upgrade to cats3 for workbench-libs compatibility
Updates workbench libs 
Optimize imports across rawls (except for XMLs, which we do not want to alter)

  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
